vapor

```diff
@@ -18,13 +21,24 @@ if (file_exists(__DIR__.'/../../autoload.php')) {
     require __DIR__.'/vendor/autoload.php';
 }
 
-if (file_exists(__DIR__.'/.env')) {
-    Dotenv::create(
-        __DIR__,
-        '.env',
-        new DotenvFactory([new EnvConstAdapter, new ServerConstAdapter])
-    )->load();
-}
+/**
+ * Load the environment variables.
+ */
+(function () {
+    if (class_exists(RepositoryBuilder::class)) {
+        $adapters = [new V4EnvConstAdapter, new V4ServerConstAdapter];
+        $repository = RepositoryBuilder::create()
+            ->withReaders($adapters)
+            ->withWriters($adapters)
+            ->immutable()
+            ->get();
```

Comment:
`get` doesn't exist? Is it supposed to be `make`?

Reply:
Yes.